File: 1-two-sum/two-sum.java
import java.util.Map;
import java.util.HashMap;
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer, Integer> auxMap = new HashMap<>();
        for(Integer i=0; i<nums.length; i++){
            if (auxMap.containsKey(target - nums[i])){
                return new int[] {auxMap.get(target - nums[i]), i};
            }
            auxMap.put(nums[i], i);
        }
        return new int[]{};
    }
}

File: 1-two-sum/two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {

    for(let index1 in nums){
        for(let index2 in nums){
                if (index1==index2) continue;
                if (nums[index1] + nums[index2] === target) return [index1, index2];
        }
    }
} ;    
